Description: Upstream changes introduced in version 1.4.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 google-mock (1.4.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #575953)
 .
 The person named in the Author field signed this changelog entry.
Author: Fredrik Hallenberg <hallon@debian.org>
Bug-Debian: http://bugs.debian.org/575953

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: google-mock-1.7.0/scripts/generator/gmock_gen.py
===================================================================
--- google-mock-1.7.0.orig/scripts/generator/gmock_gen.py	2013-07-27 18:51:26.000000000 +0000
+++ google-mock-1.7.0/scripts/generator/gmock_gen.py	2013-07-27 18:51:33.000000000 +0000
@@ -22,8 +22,7 @@
 import sys
 
 if __name__ == '__main__':
-  # Add the directory of this script to the path so we can import gmock_class.
-  sys.path.append(os.path.dirname(__file__))
+  sys.path.append("/usr/share/gmock")
 
   from cpp import gmock_class
   # Fix the docstring in case they require the usage.
